test/specs/anInitialSuite.e2e.ts

```diff
@@ -0,0 +1,144 @@
+suites run in alphabetical order, this suite has been named so it runs first.
+
+Please note that none of the other suites depend on this suite to run, it's just that if this
+suite does run, it needs to run first.
+*/
+
+describe('An Initial Suite', async () => {
+  let scratchOrg: ScratchOrg;
+
+  step('Verify our extensions are not initially loaded', async () => {
+    const workbench = await browser.getWorkbench();
+    await utilities.runCommandFromCommandPrompt(workbench, 'Developer: Show Running Extensions', 2);
+
+    const extensionNameDivs = await $$('div.name');
+    let sfdxKeywordWasFound = false;
+    let salesforceKeywordWasFound = false;
+    for (const extensionNameDiv of extensionNameDivs) {
+      const text = await extensionNameDiv.getText();
+
+      if (text.includes('sfdx')) {
+        sfdxKeywordWasFound = true;
+        utilities.log(`AnInitialSuite - extension ${text} was present, but wasn't expected`);
+      } else if (text.includes('salesforce')) {
+        if (text !== 'salesforce.system-tests') {
+          // salesforce.system-tests is expected, anything else is an issue.
+          salesforceKeywordWasFound = true;
+          utilities.log(`AnInitialSuite - extension ${text} was present, but wasn't expected before the extensions loaded`);
+        }
+      }
+    }
+    expect(sfdxKeywordWasFound).toBe(false);
+    expect(salesforceKeywordWasFound).toBe(false);
+  });
+
+  step('Verify the default SFDX commands are present when no project is loaded', async () => {
+    const workbench = await browser.getWorkbench();
+    const prompt = await utilities.openCommandPromptWithCommand(workbench, 'SFDX:');
+
+    let quickPicks = await prompt.getQuickPicks();
+    let unexpectedSfdxCommandWasFound = false;
+    for (const quickPick of quickPicks) {
+      const label = await quickPick.getLabel();
+      switch (label) {
+        // These three commands are expected to always be present,
+        // even before the extensions have been loaded.
+        case 'SFDX: Create and Set Up Project for ISV Debugging':
+        case 'SFDX: Create Project':
+        case 'SFDX: Create Project with Manifest':
+          break;
+
+        default:
+          // And if any other SFDX commands are present, this is unexpected and is an issue.
+          unexpectedSfdxCommandWasFound = true;
+          utilities.log(`AnInitialSuite - command ${label} was present, but wasn't expected before the extensions loaded`);
+          break;
+      }
+    }
+    expect(unexpectedSfdxCommandWasFound).toBe(false);
+
+    // Escape out of the pick list.
+    await prompt.cancel();
+  });
+
+  step('Set up the testing environment', async () => {
+    scratchOrg = new ScratchOrg('AnInitialSuite', false);
+    // Don't call scratchOrg.setUp() b/c we don't need to authorize a scratch org,
+    // just call setUpTestingEnvironment() and createProject().
+    await scratchOrg.setUpTestingEnvironment();
+    await scratchOrg.createProject('Developer');
```

Comment:
This is an example of where testSetup would make more sense.  A scratch org is not being authorized but the ScratchOrg class is being used to setup the testing env and to create a project. 

  ```suggestion
  testSetup = new TestSetup('AnInitialSuite', false);
    await testSetup.createProject('Developer');
    etc.
```

Reply:
Done (ScratchOrg was renamed to TestSetup)

Reply:
Awesome. Looks great.  Thanks, Jeff!